Makefile

```diff
@@ -53,7 +53,7 @@ roxygenize: DESCRIPTION hyperSpec/R/*.R
 	@echo "Roxygenize"
 	@Rscript --vanilla -e "library (methods, quietly=TRUE, verbose = FALSE); library (devtools, quietly=TRUE, verbose = FALSE); document ('hyperSpec')"
 
-DESCRIPTION: $(shell find hyperSpec -maxdepth 1 -daystart -not -ctime 0 -name "DESCRIPTION") #only if not modified today
+DESCRIPTION: $(shell find hyperSpec -maxdepth 1 -not -ctime 1 -name "DESCRIPTION") #only if not modified in the past 24 hrs
```

Comment:
It works under Ubuntu without any issues!

Reply:
Still having trouble...
<img width="1440" alt="Screen Shot 2020-05-12 at 4 20 23 PM" src="https://user-images.githubusercontent.com/31193655/81747021-07b6fb80-946d-11ea-9418-7851a1e7e6b5.png">

Reply:
@eoduniyi what command did you run? `make bootstrap` or `make`? Did you get all large files with `git lfs pull` before?